setup.py

```diff
@@ -43,6 +43,11 @@
     packages=setuptools.find_packages("src"),
     package_dir={"": "src"},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
+    entry_points={
+        "ert": [
+            "WfInitializeCase = fmu.dataio.scripts.initialize_case",
```

Comment:
Sure you want camel-case in the entry-point?

Reply:
No - my assumption was that this needed to match the class name...

Reply:
And give some thought to the actual entry-point, which is to live in the Komodo namespace. Should it have something with dataio/sumo something in its name? Should it eventually grow into a bigger tool where you should add sub-commands instead of adding more and more entry points?

Reply:
Just tested and looks like there is no link between them, so I will change to non-camel case. Any good principles to follow when naming it?

Reply:
Renamed the entry point. May trigger more renaming down the chain ref comment on "initialize_case" and possible collision with ERT context(s).

Reply:
@berland I missed your (good) comment regarding the actual entry-point. It should probably/definitely have something more generic in its name. I assume this lives in the ERT namespace, so `ert.xyz`? In that case, should we go for `ert.fmudataio`? (Just `dataio` becomes too generic in my opinion.) Other opinions and/or gut feelings? @jcrivenaes